new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Helm.java

```diff
@@ -7,31 +7,86 @@
 
 public class Helm {
 
-    public static boolean addRepo(String repoUrl) {
-        // do something!!
+    private String chart;
+    private String name;
+    private String namespace;
+    private String repoUrl;
+    private HashMap<String, String> values;
+
+    private Helm(HelmBuilder builder) {
+        this.chart = builder.chart;
+        this.name = builder.name;
+        this.namespace = builder.namespace;
+        this.repoUrl = builder.repoUrl;
+        this.values = builder.values;
+    }
+
+    public String getChart() {
+        return chart;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getNamespace() {
+        return namespace;
+    }
+
+    public String getRepoUrl() {
+        return repoUrl;
+    }
+
+    public HashMap<String, String> getValues() {
+        return values;
+    }
+
+    public boolean install(){
         return true;
     }
 
-    public static boolean installRelease(
-            String chart,
-            String name,
-            HashMap<String, String> values
-    ) {
-        // do something !!!
+    public boolean upgrade(){
         return true;
     }
 
-    public static boolean upgradeRelease(
-        String chart,
-        String name,
-        HashMap<String, String> values
-    ) {
-        // do something !!!
+    public boolean delete(){
         return true;
     }
 
-    public static boolean deleteRelease(String name) {
+    public boolean addRepo() {
         return true;
     }
 
+    public static class HelmBuilder {
```

Comment:
do we need the builder? i did a lot of googling and i did not come up with a definitive answer. this seems to be builder pattern, which is almost the same as fluent, but not exactly. i think fluent is more of a static that manages the instance. i don't mind really - i will defer to @rjeberhard and @ddsharpe on this

Reply:
fluent example https://github.com/kubernetes-client/java/blob/master/examples/src/main/java/io/kubernetes/client/examples/FluentExample.java @rjeberhard suggested uses builder, thought we wanted to use that pattern. Not sure if he meant to use that only for Operator or Domain.